Cargo.toml

```diff
@@ -115,7 +115,7 @@ turborepo = { path = "crates/turborepo" }
 turborepo-api-client = { path = "crates/turborepo-api-client" }
 turborepo-ffi = { path = "crates/turborepo-ffi" }
 turborepo-fs = { path = "crates/turborepo-fs" }
-turborepo-lib = { path = "crates/turborepo-lib" }
```

Comment:
Let's move feature changes that aren't required to fix the `yarn` issue to a separate PR